### PR TITLE
Design History log page

### LINKED ISSUE #13: Design Auth History view component

### PR DESCRIPTION
**Pull request by anthonyminyungi**
- It closes #13

### CODE REVIEW

File: src/components/history/index.tsx [hunk 1]
```diff
@@ -1,15 +1,52 @@
...
+                                </ListItemAvatar>
+                                <ListItemText 
+                                    primary={history.type} 
+                                    secondary={`${new Date(history.createdAt).toDateString()}  ${history.origin}`} 
+                                />
+                            </ListItem>
+                        )
+                    })
+                }
+            </List>
```
Comment (anthonyminyungi) [inline src/components/history/index.tsx:49]: Appearance.
스크린샷 2020-09-03 오전 2 30 22

File: src/components/main/index.tsx [hunk 2]
```diff
@@ -237,6 +235,44 @@ const dummyMemberships = [
...
 
+const dummyHistoryLogs = [
+    {
+        type: 'AUTHENTICATION',
+        createdAt: '2020-09-01',
+        origin: 'https://www.naver.com',        
+    },
+    {
+        type: 'REGISTRATION',
+        createdAt: '2020-08-31',
```
Comment (anthonyminyungi) [inline src/components/main/index.tsx:246]: Dummy data for the history logs.

File: src/components/main/index.tsx [hunk 3]
```diff
@@ -283,10 +319,14 @@ export default function MainPage({ title, favicon, currentInfos, signature }: Ma
...
-                    <Button className={classes.seeMore} startIcon={<FullscreenIcon />}>
+                    <Button className={classes.seeMore} startIcon={<FullscreenIcon />} onClick={handleClickOpen}>
                         Full Screen
                     </Button>
-                    <HistoryListPage />
+                    <Dialog open={open} fullScreen onClose={handleClose} TransitionComponent={Transition}>
+                        <DialogHeader onClose={handleClose} />
+                        <HistoryListPage histories={dummyHistoryLogs} />
+                    </Dialog>
+                    <HistoryListPage histories={dummyHistoryLogs} />
```
Comment (anthonyminyungi) [inline src/components/main/index.tsx:329]: Full screen appearance. 
스크린샷 2020-09-03 오전 2 30 58

**Review by anthonyminyungi — COMMENTED** [review]
Self-review.

File: src/components/main/index.tsx (hunk 3 above)
Comment (HyeockJinKim) [reply src/components/main/index.tsx:329]: It's great!

File: src/components/history/index.tsx (hunk 1 above)
Comment (HyeockJinKim) [reply src/components/history/index.tsx:49]: Looks good enough in my opinion.

**Review by HyeockJinKim — APPROVED** [review]
LGTM

**Pull request merged**